src/SigasiProjectCreator/CsvParser.py

```diff
@@ -17,17 +17,23 @@ def __init__(self):
 
     def parse_file(self, csv_file, options=None):
         library_mapping = dict()
+        includes = set()
+        defines = []
         with open(csv_file, 'r') as f:
             reader = csv.reader(f, skipinitialspace=True)
             for row in reader:
                 if row:
                     library = row[0].strip()
                     path = pathlib.Path(row[1].strip()).absolute().resolve()
```

Comment:
Could this not crash on some macro definitions? I'd prefer only parsing this as path when we're sure it's supposed to be a path.

Reply:
Code restructured to only run pathlib code on an expected path